dealership/vehicles.py

```diff
@@ -1,15 +1,36 @@
 class Vehicle(object):
     def __init__(self, maker, model, year, base_price, miles):
-        pass
+        self.maker = maker
+        self.model = model
+        self.year = year
+        self.base_price = base_price
+        self.miles = miles
 
 
 class Car(Vehicle):
-    pass
-
+    def sale_price(self):
+        return self.base_price * 1.2
```

Comment:
I like that your code is concise, but you could make your code more reusable by declaring the multipliers as constants instead of directly plugging them into the methods.